pandas/io/formats/format.py

```diff
@@ -625,7 +625,7 @@ def to_string(self):
                 max_len += size_tr_col  # Need to make space for largest row
                 # plus truncate dot col
                 dif = max_len - self.w
-                adj_dif = dif
+                adj_dif = dif + 1  # see GH PR #17023
```

Comment:
can you put it on the line above?

Reply:
You mean

```
dif = max_len - self.w  # see GH PR #17023
adj_dif = dif
```
?

`dif` is never used so we might as well skip it completely.

Reply:
No, I just meant to put the comment on its own line, not on the same line after the code, like

```
# '+ 1' to avoid too wide repr (GH PR #17023)
adj_dif = dif + 1
```

Reply:
I'm sorry, of course, I'll change that.